test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala

```diff
@@ -121,9 +121,33 @@ class IntentionToControlP5ControllerSpec extends SpecBase with AppWithDefaultMoc
           val view = injector.instanceOf[IntentionToControlP5View]
 
           contentAsString(result) mustEqual
-            view(intentionToControlP5ViewModel, departureIdP5, customsOfficeContactViewModel)(request, messages).toString
+            view(intentionToControlP5ViewModel, departureIdP5, messageId, customsOfficeContactViewModel)(request, messages).toString
       }
     }
-  }
 
+    "must redirect to Presentation notification frontend" in {
+      forAll(listWithMaxLength[RequestedDocumentType]()) {
+        requestedDocuments =>
```

Comment:
Very minor but the just the arbitrary CC060Type should suffice here as there's no conditional logic on the requested documents in the onSubmit